public/apps/account/tenant-switch-panel.tsx

```diff
@@ -297,8 +301,8 @@ export function TenantSwitchPanel(props: TenantSwitchPanelProps) {
 
           <EuiButton
             data-test-subj="confirm"
-            fill
-            disabled={!isMultiTenancyEnabled}
+            fill={isMultiTenancyEnabled && !invalidCustomTenant}
```

Comment:
Hi @cwperks, thanks for the review! I talked to @shanilpa, he suggested me to change the disabled button to the ghosted disabled without the _fill_ option, because that's the pattern we are moving forward with. That's why I added the condition to the _fill_ option